doc/source/whatsnew/v0.24.0.txt

```diff
@@ -487,6 +487,88 @@ Previous Behavior:
         0
     0 NaT
 
+.. _whatsnew_0240.api.dataframe_cmp_broadcasting:
+
+DataFrame Comparison Operations Broadcasting Changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Previously, the broadcasting behavior of :class:`DataFrame` comparison
+operations (``==``, ``!=``, ...) was inconsistent with the behavior of
+arithmetic operations (``+``, ``-``, ...).  The behavior of the comparison
+operations has been changed to match the arithmetic operations in these cases.
+(:issue:`22880`)
+
+The affected cases are:
+  - operating against a 2-dimensional ``np.ndarray`` with either 1 row or 1 column
```

Comment:
Can you indicate here what changes for those cases?

Reply:
Will update

Reply:
Updated; I think other whatsnew comments addressed.  Note that Previous Behavior blocks refer to 0.23, _not_ pre-this-PR